src/scss/includes/direction-panel.scss

```diff
@@ -36,6 +36,8 @@
       position: fixed;
       top: 64px;
       width: 100vw;
+      background-color: $background;
+      overflow-y: scroll;
```

Comment:
Out of curiosity, why is this explicit "scroll" required, instead of "auto" ?

Reply:
You're right it's not, I guess I got heavy-handed while battling with Safari :) I'll put `auto` instead.